### PR TITLE
Options in schema validation don't have to be an array

### PR DESCRIPTION
**Pull request by pascalopitz**
To facilitate:

    optional: {
        options: { checkFalsy: true }
    },

as an alias of

    optional: {
        options: [{ checkFalsy: true }]
    },

### CODE REVIEW

File: lib/express_validator.js [hunk 1]
```diff
@@ -265,9 +265,18 @@ function validateSchema(schema, req, loc, options) {
...
 
+    var opts;
+
     for (var methodName in schema[param]) {
       validator.failMsg = schema[param][methodName].errorMessage || paramErrorMessage || 'Invalid param';
-      validator[methodName].apply(validator, schema[param][methodName].options);
+
+      opts = schema[param][methodName].options;
+
+      if (opts && !Array.isArray(opts)) {
```
Comment (rustybailey) [inline lib/express_validator.js]: Let's be more explicit here and check for something like `opts != null` instead of just implicitly checking `opts`.

**Comment by rustybailey** [comment]
In the README description for Validation by Schema, I would have a sentence or two explaining how to use the options property properly. Default is have an array of params, but if you only have one param, you don't need to put it in an array.

**Comment by gustavohenke** [comment]
Thank you @pascalopitz and sorry for taking this long to accept!

**Comment by gustavohenke** [comment]
Hi! This is available as of v2.21.0!

**Pull request merged**